substrate/frame/support/src/lib.rs

```diff
@@ -819,6 +819,8 @@ macro_rules! assert_ok {
 ///
 /// This macro is intended to be used in conjunction with `tt_call!`.
 #[macro_export]
+#[doc(hidden)]
+#[deprecated(note = "This macro was intended to be used internally. It will be removed in 2025")]
```

Comment:
we can also remove it IMO

Reply:
yeah just remove it.

It would only break someone who is somehow using this intentially internal macro?

Seems like it would be zero people.

Reply:
I agree, removed in https://github.com/paritytech/polkadot-sdk/pull/5355/commits/3397dbdcb422f4983704ef4e3d68d0a3eae2e6f7